src/components/ExportDataModal.js

```diff
@@ -17,9 +17,24 @@ import * as Utils from 'src/libs/utils'
 import validate from 'validate.js'
 
 
-const InfoTile = ({ infoStyle, content, iconName }) => {
-  return div({ style: { ...infoStyle, display: 'flex', alignItems: 'center' } }, [
-    icon(iconName, { size: 36, style: { flex: 'none', marginRight: '0.5rem' } }),
+const warningStyle = {
+  border: `1px solid ${colors.warning(0.8)}`, borderLeft: 'none', borderRight: 'none',
+  backgroundColor: colors.warning(0.15),
+  padding: '1rem 1.25rem', margin: '0 -1.25rem',
+  fontWeight: 'bold', fontSize: 12
+}
+const errorStyle = {
+  ...warningStyle,
+  border: `1px solid ${colors.danger(0.8)}`,
+  backgroundColor: colors.danger(0.15)
+}
+
+const InfoTile = ({ isError = false, content }) => {
+  const [style, shape, color] = isError ?
+    [errorStyle, 'error-standard', colors.danger()] :
+    [warningStyle, 'warning-standard', colors.warning()]
```

Comment:
```suggestion
    [warningStyle, 'warning-standard', colors.warning()]

```

Reply:
I'm all about readability, but I don't see a difference here. Is it indentation that github is somehow hiding?
<img width="486" alt="Screen Shot 2020-05-07 at 6 54 29 PM" src="https://user-images.githubusercontent.com/19228696/81352366-5ee75580-9094-11ea-8a88-9a6ccd49f5ff.png">